ros/node.go

```diff
@@ -15,6 +14,8 @@ import (
 	"strings"
 	"sync"
 	"time"
+
+	"github.com/edwinhayes/rosgo/xmlrpc"
```

Comment:
Can you find out whether this change was just accidental, or whether the IDE and Go tools did this automatically?